### PR TITLE
isort: Add to CI; fix some files

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by odscjames** [comment]
Broken isort - https://github.com/ThreeSixtyGiving/datastore/runs/1429865942?check_suite_focus=true

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -22,3 +22,4 @@ jobs:
     - run: flake8
     # To avoid datagetter errors, we must specifically exclude src/datagetter
     - run: black --check --exclude "/(\.eggs|\.git|_cache|src\/datagetter)/" ./
+    - run: cd datastore && isort --check-only *.py */
```
Comment (michaelwood) [inline .github/workflows/lint.yml:25]: If you run isort from the top level directory you get one output, if you run it from datastore sub-directory as you're doing you get another output. So you'd also need to `cd` into all the relevant directories for isort not to go wrong? (you're not checking tools directory for instance but python-black and flake8 are).

e.g.
```
$ isort ./datastore/
$ git diff --stat
38 files changed, 38 insertions(+), 73 deletions(-)
```

```diff
@@ -1,9 +1,8 @@
+import additional_data.models as db
 from django.contrib import admin
 from django.contrib.postgres.fields import JSONField
 from prettyjson import PrettyJSONWidget
 
-import additional_data.models as db
-
 
 class JSONFieldAdmin(admin.ModelAdmin):
     list_per_page = 20
```

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -22,3 +22,4 @@ jobs:
     - run: flake8
     # To avoid datagetter errors, we must specifically exclude src/datagetter
     - run: black --check --exclude "/(\.eggs|\.git|_cache|src\/datagetter)/" ./
```
Comment (michaelwood) [inline .github/workflows/lint.yml:24]: Maybe a better way would be not to install all the other dependencies and just select the lint packages. e.g.

```yaml
- run: pip install `/bin/grep "black==" ./requirements_dev.txt`
- run: black . --check
```

Should be a bit faster too.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (odscjames) [reply .github/workflows/lint.yml:25]: Sounds like something very odd is happening with Python package dirs here I'd like to dig into. How many "relevant directories" are there?

Comment (odscjames) [reply .github/workflows/lint.yml:25]: It appears to just be 2 - having 2 isort lines, 1 for each dir sounds fine?

File: .github/workflows/lint.yml (hunk 2 above)
Comment (odscjames) [reply .github/workflows/lint.yml:24]: A good general point, but this applied before and it applies to many of our lint runs so I'd request it's for a different pull request.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (michaelwood) [reply .github/workflows/lint.yml:25]: Sounds awkward! for the very reason you did `--exclude` on python-black for not wanting to have to maintain a list of places to check.

File: .github/workflows/lint.yml (hunk 2 above)
Comment (michaelwood) [reply .github/workflows/lint.yml:24]: Ah didn't notice this wasn't part of the PR, my mistake.

**Comment by michaelwood** [comment]
I ran latest version of isort again today to see if it had changed behaviour, in my opinion it doesn't improve readability of the code when it groups imports together. I have a strong preference for logical groupings of imports over alphabetical/+group.

**Pull request closed without merging**